optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/result/BenchmarkResultIO.java

```diff
@@ -92,6 +91,16 @@ public void writePlannerBenchmarkResult(File benchmarkReportDirectory,
         return plannerBenchmarkResultList;
     }
 
+    private File[] retrieveSortedBenchmarkReportDirectories(File benchmarkDirectory) {
+        File[] benchmarkReportDirectories = benchmarkDirectory.listFiles((FileFilter) DirectoryFileFilter.INSTANCE);
+        if (benchmarkReportDirectories == null) {
+            return new File[] {};
```

Comment:
Unconventional space.

Reply:
Another proof the checkstyle + KIE conventions is necessary. Thanks, will fix